packaging/opensuse-42.2/snapd.spec

```diff
@@ -149,6 +148,12 @@ go install -s -v -p 4 -x -tags withtestkeys github.com/snapcore/snapd/cmd/snapd
 %gobuild cmd/snapctl
 %gobuild cmd/snap-update-ns
 
+# Removing static link to libseccomp due to libseccomp.a is not available in opensuse
+# This is not affecting due to seccomp is disabled for opensuse
```

Comment:
@mvo5 ok, I got confused because of this "configure" line in the spec
# NOTE: until snapd and snap-confine have the improved communication mechanism
# we need to disable apparmor as snapd doesn't yet support the version of
# apparmor kernel available in SUSE and Debian. The generated apparmor profiles
# cannot be loaded into a vanilla kernel. As a temporary measure we just switch
# it all off.
%configure --disable-apparmor --disable-seccomp --libexecdir=/usr/lib/snap

Is it ok, or we should remove "--disable-seccomp"